### ISSUE TITLE
Better error handling for gzip exceptions

### ISSUE THREAD

**Issue by chrisbcn** [avocode/avocode#447]
IMPORTANT: Please try to describe how did it happen.

-----
App v.2.4.2-d618096
Source UID: 87446
```
Error: invalid stored block lengths
    at Zlib._handle.onerror (zlib.js:345:17)
```

**Comment by petrbrzek** [comment]
Thanks for your report. Does this error happen regularly or rather rarely? We definitely have to handle this error better.

**Comment by rychlis** [comment]
Another related error:
```
Error: incorrect header check
    at Zlib._handle.onerror (zlib.js:345:17)
```

**Comment by petrbrzek** [comment]
This error should be now fixed with the latest Avocode 2.5.3. Please try it and let us know.
